src/components/Home/Home.tsx

```diff
@@ -61,7 +62,7 @@ function Home() {
             yourself to put your skills to the test.
           </p>
           <p>
-            <button className="button-main m-t-10">Read more</button>
+            <Link to="/timathon"><button className="button-main m-t-10">Read more</button></Link>
```

Comment:
remove button since you added a `<Link>` ?

Reply:
If i removed the button, there would be nowhere to click the link. Can you specify where you want me to add this link to please.

Reply:
add the styles to `<Link>` and the text inside of it instead of inside `<button>` like this ⬇️
```html
<Link to="/timathon" className="button-main m-t-10">Read more</Link>
```

Reply:
Okay, will do.